api/proto/teleport/legacy/client/proto/authservice.proto

```diff
@@ -2296,8 +2296,83 @@ message ExportUpgradeWindowsResponse {
   string SystemdUnitSchedule = 3;
 }
 
+message AssistantRequest {
+  // conversation_id it's the ID of a conversation.
```

Comment:
Nit: These comments will be applied to generated Go structs so they should follow Go field naming conventions. Same everywhere below.
```suggestion
  // ConversationID it's the ID of a conversation.
```

Reply:
The field is called `ConversationId`, but I get your point. https://github.com/gravitational/teleport/blob/e87cb3b2cae8bdfc8cd9bb56cd1586876a3bb1f4/api/client/proto/authservice.pb.go#L14052
We should probably stop using Go way of commenting on protobufs. If we ever generate a client in a different language all comments will be wrong.